RecoLocalTracker/SiPixelRecHits/plugins/PixelCPEGenericESProducer.cc

```diff
@@ -98,4 +96,25 @@ std::unique_ptr<PixelClusterParameterEstimator> PixelCPEGenericESProducer::produ
                                            lorentzAngleWidthProduct);
 }
 
+void PixelCPEGenericESProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+
+  // from PixelCPEBase
+  PixelCPEBase::fillPSetDescription(desc);
+
+  // from PixelCPEGeneric
+  PixelCPEGeneric::fillPSetDescription(desc);
+
+  // specific to PixelCPEGenericESProducer
+  desc.add<std::string>("ComponentName", "PixelCPEGeneric");
+  desc.add<edm::ESInputTag>("MagneticFieldRecord", edm::ESInputTag(""));
+  desc.add<bool>("useLAAlignmentOffsets", false);
+
+  // vestigial (present in certain HLT menus)
+  desc.addOptional<double>("TanLorentzAnglePerTesla", 0.106);
+  desc.addOptional<std::string>("PixelErrorParametrization", "NOTcmsim");
```

Comment:
```suggestion
  desc.addOptional<std::string>("PixelErrorParametrization");
```
assuming this one is also obsolete.